big_tests/tests/push_http_SUITE.erl

```diff
@@ -165,6 +165,12 @@ stop_pool() ->
     ejabberd_node_utils:call_fun(mongoose_wpool, stop, [http, <<"localhost">>, http_pool]),
     ok.
 
+http_notifications_port() ->
+    ct:get_config({hosts, mim, http_notifications_port}).
+
+http_notifications_host() ->
+    "http://localhost:" ++ integer_to_list(http_notifications_port()).
+
```

Comment:
This is duplicated between `mod_http_notification_SUITE` and `push_http_SUITE`, wouldn't it make sense to move it to some helper module? Not like crucial, but it wouldn't hurt, right?

Reply:
Done

[configure-ports-using-test-config 3c61bf0e0] Move http_notifications_port/0, http_notifications_host/0 into push_helper
 3 files changed, 12 insertions(+), 12 deletions(-)